src/toil/jobStores/fileJobStore.py

```diff
@@ -187,6 +191,43 @@ def fileExists(self, jobStoreFileID):
             raise NoSuchFileException("Path %s is not a file in the jobStore" % jobStoreFileID)
         return True
 
+    def importFile(self, sourceUrl, jobStoreID=None):
+        url = urlparse.urlparse(sourceUrl)
+        if url.scheme.lower() == 'file':
```

Comment:
not as big of a deal here but these checks should also use regex's for consistency